tests/python/unittest/test_tir_intrin.py

```diff
@@ -303,11 +305,11 @@ def test_binary_search():
 
 
 if __name__ == "__main__":
-    test_nearbyint()
-    test_unary_intrin()
-    test_round_intrinsics_on_int()
-    test_binary_intrin()
-    test_ldexp()
-    test_clz()
-    test_fma()
+    # test_nearbyint()
+    # test_unary_intrin()
+    # test_round_intrinsics_on_int()
+    # test_binary_intrin()
+    # test_ldexp()
+    # test_clz()
+    # test_fma()
```

Comment:
You may uncomment these lines back :eyes:?

Reply:
Yes, but that will not influence unittest because pytest picks all functions start with `test_` and ignores all lines below `if __name__ == '__main__'`, which is only for debugging.